example/blog/tests.py

```diff
@@ -17,7 +17,7 @@ def setUpClass(cls):
 
     def create_article(self, title: str, excerpt: str | None, body: str) -> Article:
         """Create a Article with the given informations."""
-        body = Document.objects.create(content="test body")
+        body = Document.objects.create(content=body)
```

Comment:
We should also rename this variable, in order to also be type-correct (`body_document` sounds like a good name)